pkg/slayers/path/scion/raw_spec.gobra

```diff
@@ -125,6 +135,22 @@ pure func (s *Raw) ValidCurrIdxs(ghost ub []byte) bool {
 		s.Base.ValidCurrIdxs()
 }
 
+ghost
+requires acc(s.Mem(ub), _)
+decreases
+pure func (s *Raw) InfsMatchHfs(ghost ub []byte) bool {
+	return unfolding acc(s.Mem(ub), _) in
+		s.Base.InfsMatchHfs()
+}
+
+ghost
+requires acc(s.Mem(ub), _)
+decreases
+pure func (s *Raw) CurrInfMatchesCurrHF(ghost ub []byte) bool {
+	return unfolding acc(s.Mem(ub), _) in
+		s.Base.GetBase().CurrInfMatchesCurrHF()
+}
```

Comment:
As a general comment, this is fine but I plan to deprecate all these wrappers. First, they hide the fact that these functions only mention fields in the Base. Second, to know the definitions, we need to follow a few layers of wrappers, which is often frustrating. Instead, in #347, I will drop all of these, introduce a GetBase() function for `*Raw` and the other types, and write all specs like `s.GetBase(ub).CurrInfMatchesCurrHF()`. While this is longer, it addresses the two points I mentioned